lib/models/purchases_configuration.dart

```diff
@@ -23,6 +23,10 @@ class PurchasesConfiguration {
   /// use standardUserDefaults.
   String? userDefaultsSuiteName;
 
+  /// iOS-only, will be ignored for Android.
+  /// Set this to FALSE to disable StoreKit2.
```

Comment:
Maybe add `/// Default is true.` as the other comments? It's kind of implicit by this line, but I think it's good to be explicit.

Reply:
Done.